mashafrancis/butternut-user
# chore(setup): setup project folder and files

#### Description
This sets up the project folder for the project butternut

#### Type of change
<pre>
+ Added typescript, tslinting, and tsconfig files
+ Added readme.md file
+ Added package.json file and node modules
+ Added graphql and its dependancies
</pre>

#### How Has This Been Tested?
1. Run `yarn start:dev`
2. Open your browser and run the following link `http://localhost:4000/`
3. Create a query { hello } and run to see the response

#### Story ID
#169730577
